global.yaml

```diff
@@ -253,6 +262,7 @@ global:
     #   memory: 128Mi
 
   defaultPodDisruptionBudget:
+    enabled: true
```

Comment:
Not sure if that is relevant, but when we deploy pilot with only one replica, we probably don't want to have a pdb. If I am not mistaken we block node upgrades otherwise.

Reply:
That's easy - if replica == 1 .... 

I don't have a problem to add it - but redundant flags that are not easy to explain are not ideal.
At least if we keep it - add a doc explaining when to turn it off and why.  I don't know, and I suspect many users will not know either.

Reply:
This is in 1.1 so I think we need it but we can deprecate/document it

Reply:
PDBs were added to help with in-place upgrades.  They can probably be completely removed in the future prior to 1.2.

Reply:
I think using PDB makes sense in any case to guarantee HA even out of the context of installation.
Real HA of e.g. Pilot would require 3 replica and a PDB of 2, right? Otherwise, during a node upgrade there could be only one pod left, that'd be the perfect time for that one to fail...